benchmark/httpapi.py

```diff
@@ -114,10 +137,11 @@ def post(self, request):
         request.setHeader(b'content-type', b'application/json')
         try:
             json = loads(request.content.read())
+            timestamp_parser.parse(json['timestamp'])
+        except KeyError as e:
+            raise BadRequest("'{}' is missing".format(e.message))
```

Comment:
I didn't see any new tests for this new failure path, could we add a test for this? I was trying to verify my comment on line 137 and whipped up https://github.com/ClusterHQ/benchmark-server/pull/4 which surprised me when it failed.

My apologies if this is just my error and everything is fine, but that looks to me like there might be some underlying infrastructure that is not working as it should.